packages/manfred2html/src/engine/html-parts/studies-section/studies-section.ejs

```diff
@@ -0,0 +1,51 @@
+<%_ if (studiesCollection && studiesCollection.length !== 0) { -%>
+    <section class="main__section">
+      <div class="main__section__header">
+        <h2>Educación</h2>
+        <hr class="secondary-divider" />
+      </div>
+      <%_ for (const study of studiesCollection) { -%>
+      <div class="main__section__item">
+        <div class="date date-desktop">
+          <p>Inicio: <b><%- study.startDate %></b></p>
+          <p>Fin: <b><%- study.finishDate %></b></p>
```

Comment:
Only display the finish date if we have a date. Review the experience section for comparison